apps/dev/src/combobox/combobox-demo.component.scss

```diff
@@ -0,0 +1,11 @@
+.oversized-option {
+  line-height: 40px;
+
+  &:nth-child(n + 1) {
+    background-color: #ff8 !important;
```

Comment:
Aaah interesting, didn't know that!
I usually avoid `!important` whenever possible, but since this is just in the dev app I thought it didn't matter.

Reply:
please remove even in the dev-app as we don't want to have anywhere an example where it could be copied.